src/locales/en.json

```diff
@@ -31,16 +31,16 @@
       "after_created": "Once your Note is created, you can find it in your "
     },
     "SavingIndicator": {
-      "saved_just_now": "Saved just now.",
-      "saved_secs_ago": "Saved %{time} sec ago",
-      "saved_min_ago": "Saved %{time} min ago",
-      "saved_mins_ago": "Saved %{time} min ago",
-      "saved_hour_ago": "Saved %{time} hour ago",
-      "saved_hours_ago": "Saved %{time} hours ago",
-      "saved_day_ago": "Saved %{time} day ago",
-      "saved_days_ago": "Saved %{time} days ago",
+      "saved_just_now": "Last update now",
```

Comment:
Shouldn't it be `Last updated`? Or `Last update: now` or `last update was now`, but `Last update now` seems wrong.

Reply:
not as it is requested : https://trello.com/c/T1FZP8U9/132-etu-jai-linformation-de-lenregistrement-de-ma-note-2j